.github/workflows/windows.yml

```diff
@@ -142,7 +142,7 @@ jobs:
 
       - name: Install libraries with vcpkg
         run: |
-          vcpkg install
+          vcpkg install --vcpkg-root=C:\Users\runneradmin\scoop\apps\vcpkg\current
```

Comment:
It can be `--vcpkg-root=%HOME%\scoop\apps\vcpkg\current` or `--vcpkg-root=%USERPROFILE%\scoop\apps\vcpkg\current`.